docs/source/outputs.rst

```diff
@@ -0,0 +1,119 @@
+
+In Stage 3:
+   - If ``isplots_S3`` = 1: ``Eureka!`` will plot the 2-dimensional, non-drift-corrected light curve. 
+
+   .. figure:: ../media/S3_out/fig3101.png
+      :alt: Stage 3 2-dimensional spectrum plot
+
+      Stage 3 output: 2-Dimensional Spectrum Plot
+
+   - If ``isplots_S3`` = 3: ``Eureka!`` will plot the results of the background and optimal spectral extraction steps for each exposure in the observation, as well as the source position on the detector.
+
+   .. figure:: ../media/S3_out/fig3301.png
+      :alt: Stage 3 background subtracted flux plot
+
+      Stage 3 output: Background Subtracted Flux Plot
+
+   .. figure:: ../media/S3_out/fig3302.png
+      :alt: Stage 3 1-dimensional spectrum plot
+
+      Stage 3 output: 1-Dimensional Spectrum Plot
+
+   .. figure:: ../media/S3_out/fig3303.png
+      :alt: Stage 3 source position fit
+
+      Stage 3 output: Source Position Fit Plot
+
+   .. figure:: ../media/S3_out/fig3305.png
+      :alt: Stage 3 weighting profile
+
+      Stage 3 output: Weighting Profile Plot
+
+   - If ``isplots_S3`` = 5: ``Eureka!`` will plot the Subdata plots from the optimal spectral extraction step.
+
+   .. figure:: ../media/S3_out/fig3501.png
+      :alt: Stage 3 subdata plot
+
+      Stage 3 output: Spectral Extraction Subdata Plot
+   
+.. _s4-out:
+
+Stage 4 Outputs
+---------------
+
+In Stage 4:
+   - If ``isplots_S4`` = 1: ``Eureka!`` will plot the spectral drift per exposure.
+
+   .. figure:: ../media/S4_out/fig4101.png
+      :alt: Stage 4 spectral drift plot
+
+      Stage 4 output: Spectral Drift Plot
+
+   - If ``isplots_S4`` = 2: ``Eureka!`` will plot the drift-corrected 2-dimensional lightcurve with extracted bins overlaid.
+
+   .. figure:: ../media/S4_out/fig4201.png
+      :alt: Stage 4 2-dimensional binned spectrum
+
+      Stage 4 output: 2-Dimensional Binned Spectrum
+
+   - If ``isplots_S4`` = 3: ``Eureka!`` will plot the spectroscopic lightcurves for each wavelength bin.
+
+   .. figure:: ../media/S4_out/fig4301.png
+      :alt: Stage 4 spectroscopic lightcurve
+
+      Stage 4 output: Spectroscopic Lightcurve
+
+   - If ``isplots_S4`` = 4: ``Eureka!`` will plot the cross-correlated reference spectrum with the current spectrum for each integration.
+
+   .. figure:: ../media/S4_out/fig4401.png
+      :alt: Stage 4 cross correlated reference spectrum
+
+      Stage 4 output: Cross-Correlated Reference Spectrum
+
+   - If ``isplots_S4`` = 5: ``Eureka!`` will plot the cross-correlation strength for each integration.
+
+   .. figure:: ../media/S4_out/fig4501.png
+      :alt: Stage 4 cross correlation strength
+
+      Stage 4 output: Cross-Correlation Strength
+
+
+.. _s5-out:
+
+Stage 5 Outputs
+---------------
+
+In Stage 5:
+   - If ``isplots_S5`` = 1: ``Eureka!`` will plot the fitted lightcurve model over the data in each channel.
+   - If ``isplots_S5`` = 2: ``Eureka!`` will plot an RMS deviation plot for each channel to help check for correlated noise.
```

Comment:
This is not correct - the fig5200 name does not mean isplots_S5>=2. It is just the second plot made in S5, and it gets called if isplots_S5>=3. I assume you'll need to update all of these notes about plots accordingly

Reply:
It's possible the intended naming convention had been what you assumed, but it is not that way at least for Stage 5 (and I think Stage 4) although I think that is how the plot names are made in Stage 3. @kevin218, we should really settle on one figure naming convention and use it throughout all stages.

Reply:
These should be fixed now. I agree the naming conventions should be consistent, and I think the way it works in Stage 3 actually works well. We might want to update the specific numbering eventually, though.a